### DISCUSSION TITLE
packaging/channels

### DISCUSSION THREAD

**Discussion by giscus** [velopack/velopack.docs#18] [General]
# packaging/channels

Channels is a fundemental part of how Velopack understands and builds releases. Every release must belong to a channel. If you do not specify a channel when building a release (via the --channel) argument, the default channel will be the name of the target Operating System (eg. win, osx, or linux).

https://docs.velopack.io/packaging/channels

**Comment by shunkica** [comment]
The overload doesn't seem to exist `new UpdateManager("https://the.place/you-store/updates", "beta")`.
The second parameter must be of type `UpdateOptions`

**Reply by shunkica** [reply]
It is unclear how we can fetch the current channel, the channel which the current app was built for.

**Reply by caesay** [reply]
Thanks for pointing out the mistake in the docs. It has been updated now, and yes you'll need to provide an `UpdateOptions` with the channel you'd like to switch to. (but you should not provide a channel if you intend to stay on the current channel). To find the current channel you can use `VelopackLocator.GetDefault().Channel`
